lib/stream.js

```diff
@@ -44,39 +44,37 @@ Stream.Stream = Stream;
 // Internal utilities
 try {
   const types = require('util').types;
-  if (types && typeof types.isUint8Array === 'function') {
-    Stream._isUint8Array = types.isUint8Array;
+  if (types && typeof types.isArrayBufferView === 'function') {
+    Stream._isArrayBufferView = types.isArrayBufferView;
   } else {
     // This throws for Node < 4.2.0 because there's no util binding and
     // returns undefined for Node < 7.4.0.
```

Comment:
This comment would need to be updated. When has `isArrayBufferView`  been added?

Reply:
v7.4.0.